markdown/events/2023/bytesize_funcscan.md

```diff
@@ -16,8 +16,67 @@ Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+[1:24](https://www.youtube.com/watch?v=c1CnE6jPhpg&t=84)
+Who would be interested in such a pipeline, which identifies these compounds? In natural product discovery, where you can identify these compounds to develop therapeutics, in antibiotic research, in environmental metagenomics, or simply to have functional and genomic annotations. In these research fields, the detection of these compounds is already being done with a couple of tools, however, there are certain issues. One of them would be the efficiency, because mostly you apply the tools manually and then you only have a very specific purpose of the tool. You can identify a single compound, but it's not very broad and you have only a single algorithm that identifies the output. It could be more feasible to have this whole process streamlined in a pipeline. Also the output of these tools is not standardized. Another issue would be the reproducibility, because throughout the years, the tools develop new functions, bugs are fixed. It's very important for researchers to record which versions of which tools they are using, which is hard if you execute them manually on your samples. Also data privacy, there are a bunch of tools that offer web services where you can upload your data where they are analyzed for you. However, this requires that you give your data to a third party, which is not always intended or even possible. Another issue is that bioinformatics skills are often needed. Sometimes you even have to write small bash scripts to execute the tools on your data, which is not possible for all people. For example if they are biochemists who just want to know what is in the data, they don't want to be trained bioinformaticians.
+
+[3:30](https://www.youtube.com/watch?v=c1CnE6jPhpg&t=210)
+These are the many problems that our pipeline tackles, namely that it is very scalable since it's a Nextflow pipeline, all nf-core pipelines are Nextflow pipelines. They are very efficient and scalable. You can execute them on your local computer, laptop up to the Institute's HPC. They are reproducible since they record all the tools and versions of the tools. Of course, you can decide where you want to have your data, you are not forced to put them on any web server. Also, it is very easy to execute the pipeline, which you will see later when we come to the tutorial part.
```

Comment:
```suggestion
These are the many problems that our pipeline tackles, namely that it is very scalable since it's a Nextflow pipeline, all nf-core pipelines are Nextflow pipelines. They are very efficient and scalable. You can execute them on your local computer, laptop, up to the institute's HPC. They are reproducible since they record all the tools and versions of the tools. Of course, you can decide where you want to have your data, you are not forced to put them on any web server. Also, it is very easy to execute the pipeline, which you will see later when we come to the tutorial part.
```